### ISSUE TITLE
procedure-specific fine-tuned models

### ISSUE THREAD

**Issue by cascat0** [SamuelSchmidgall/GSViT#3]
Hello. 

I would like fine-tuned models on laparoscopic cholecystectomy (Cholec80), and I would greatly appreciate it if you could grant me access to these models to further explore their capabilities and apply them to my work.

**Comment by cascat0** [comment]
I tried to fine-tune it on the Cholec80 dataset using the pre-trained weights you provided. But the results were terrible, with only around 40% accuracy.
